packages/react-components/src/components/MessageStatusIndicator.tsx

```diff
@@ -96,6 +97,7 @@ export const MessageStatusIndicator = (props: MessageStatusIndicatorProps): JSX.
           data-ui-id="chat-composite-message-tooltip"
           calloutProps={{ ...calloutProps }}
         >
+          <LiveMessage message={strings.failedToSendAriaLabel ?? ''} aria-live="polite" />
```

Comment:
nit: Should we only add the `LiveMessage` if we know `!!strings.failedToSendAriaLabel` is `true` to avoid an empty announcer entry?